### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by samoslag**
- Display token symbol in Activity instead of token name
- Fix bond width on Dashboard
- Implement Finalizing message in actions
- Align activity and rewards on Dashboard
- Remove blur in overlay actions for better performance
- Display balances on Dashboard over 1.000.000 in human readable format
- Restyle bottom part of Bond component
- Implement new Account Selector
- Update UI Kit to version 1.0.5
- Add ability to select different pool from specific pool page
- Extend hiding balance feature to apply to navigation as well
- Restyle main logo in navigation
- Polish animations and navigation links
- Create new Bond confirmation popup
- Restyle Token Creator (without restyled popups ATM)
- Update react-lib to version 1.2.3

### CODE REVIEW

**Comment by Frenkiee** [comment]
There are a few errors, which have to be fixed before merging.

1. Supply in the confirmation module is not looking good
Screenshot 2022-09-14 at 10 01 30

2. Supply on both sides should look the same
Screenshot 2022-09-14 at 10 01 14
Screenshot 2022-09-14 at 10 06 24

3. When clicking on the burnable switch button there is a twitch 
Screenshot 2022-09-14 at 10 00 33

4. Why do we still have the network name in our logo?
Screenshot 2022-09-14 at 09 58 24

5. Bond has an error. 
Screenshot 2022-09-14 at 09 58 06

6. When starting an app there are errors:
Screenshot 2022-09-14 at 09 57 54

7. Can not select mainnet/testnet?
Screenshot 2022-09-14 at 09 55 10

**Comment by Netherdrake** [comment]
I think "Decimal value on chain" is poor ux. Its confusing and doesnt serve the purpose of communicating the precision, so maybe just remove it.

**Comment by Frenkiee** [comment]
Few more things for the token creator. Use the same UX we have for swap/stake/unstake/send. This way we can avoid "new pages" + app gains consistency over the pages 

Screenshot 2022-09-14 at 10 15 04
Screenshot 2022-09-14 at 10 14 59

**Comment by Netherdrake** [comment]
Can we just focus on bugfixes in this PR and do another PR for token creator?

**Comment by samoslag** [comment]
Most of these aren't bugs, just update **react-lib** to version specified in **package.json** and everything should work.
Unstyled Creator components are to be included in the next merge.
Other requested changes have been completed. Should be ok to merge.

File: src/utils/displayBalance.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import BigNumber from 'bignumber.js';
+import Uik from '@reef-defi/ui-kit';
+import { Token, utils } from '@reef-defi/react-lib';
+
+export const displayBalance = (balance: string | number): string => {
+  if (typeof balance === 'string') {
+    // eslint-disable-next-line
+    balance = new BigNumber(balance).toNumber();
```
Comment (Frenkiee) [inline src/utils/displayBalance.ts:8]: here the numeric overflow can happen. Keep balance in string or in BigNumber

**Review by Frenkiee — APPROVED** [review]

**Pull request merged**